docusaurus.config.js

```diff
@@ -120,13 +120,19 @@ module.exports = {
     'react-iframe',
     ['@docusaurus/plugin-client-redirects',
       {
+        fromExtensions: ['js'], // /myPage.html -> /myPage
+        redirects: [
+          {
+            from: '/src/theme/NotFound.js',
+            to: '/',
+          },
+        ],
```

Comment:
@swiatekm-sumo I'm looking to route our 404 "Not Found" page to our homepage. Not sure if I did it right.

This is just temporary, to resolve the issue of users accessing our site through Google results and getting 404s.

cc: @SumoLogic/open-source-collection-team @stacykor

Reply:
Update - I'm going to merge this now since we've been getting feedback about this use case. If it doesn't work, I'll revert.